stanford_sites_helper.install

```diff
@@ -8,6 +8,29 @@
  */
 function stanford_sites_helper_install() {}
 
+/**
+ * Implements hook_requirements().
+ */
+function stanford_sites_helper_requirements($phase) {
+  $requirements = array();
+
+  switch ($phase) {
+    case 'runtime':
+      $t = get_t();
```

Comment:
I don't think you need to do this on runtime checks. Just install time. You should have the `t()` function available at runtime.

Reply:
I stole this from code you wrote. Blame Canada.

Reply:
With the whole #metoo movement thing going on, it does appear that one can be accountable for past mistakes even if they attempt to right past wrongs.